amr-wind/turbulence/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 target_sources(${amr_wind_lib_name} PRIVATE
-  LaminarModel.cpp
-  turb_utils.cpp
+  Smagorinsky.cpp
+  OneEqKsgs.cpp
+  AMD.cpp
+  AMDNoTherm.cpp
+    Kosovic.cpp		
```

Comment:
I think something got mixed up here. I think you have the CMakeLists file from the LES folder in the wrong place.